relay-profiling/src/android.rs

```diff
@@ -0,0 +1,92 @@
+use serde::{Deserialize, Serialize};
+
+use crate::utils::deserialize_number_from_string;
+use crate::ProfileError;
+use android_trace_log::AndroidTraceLog;
+use relay_general::protocol::EventId;
```

Comment:
Ideally, please sort these imports like that:

```suggestion
use android_trace_log::AndroidTraceLog;

use relay_general::protocol::EventId;

use crate::utils::deserialize_number_from_string;
use crate::ProfileError;
```

This also applies to the other files.